src/server/scripts/Northrend/IcecrownCitadel/boss_rotface.cpp

```diff
@@ -480,8 +480,8 @@ class spell_rotface_ooze_flood : public SpellScriptLoader
                 targets.sort(Trinity::ObjectDistanceOrderPred(GetCaster()));
 
                 // .resize() runs pop_back();
-                if (targets.size() > 4)
-                    targets.resize(4);
+                if (targets.size() > 5)
+                    targets.resize(5);
 
```

Comment:
Yes but here we try to get the 2 triggers that should cast 69783. First one is the caster of the spell, 2 & 3 are those that spawn the pools (on the ground) 4 & 5 are those that are in the air and should cast Spell 69783.
made a little image in paint that could help :p
http://puu.sh/iWGvP/9ca1f4c163.png